app/jobs/readings_job.rb

```diff
@@ -1,24 +1,22 @@
-class ReadingsToDatabaseJob < ApplicationJob
-  include RedisScanning
+class ReadingsJob < RedisSetJob
```

Comment:
I renamed this so that we wouldn't accidentally end up running jobs enqueued by the old code with the new code, since they're not compatible.